ketcher-autotests/tests/Macromolecule-editor/Sequence-Mode/sequence-mode-selection.spec.ts

```diff
@@ -204,7 +203,7 @@ test.describe('Sequence mode selection for view mode', () => {
     await takeEditorScreenshot(page);
   });
 
-  test('Select a nucleotide or a group of nucleotides, then switch to another tool (excluding Erase). Confirm that selection is cleared.', async ({
+  test('Select a nucleotide or a group of nucleotides, then click in the middle of the screen. Confirm that selection is cleared.', async ({
```

Comment:
The name of test was changed, so it now generates screenshots with new name. Please delete old screenshots then.